subgraph/subgraph.yaml

```diff
@@ -6,9 +6,9 @@ dataSources:
     name: FleekNFA
     network: mumbai
     source:
-      address: "0x34F21E970A7cd383eE429aDB5ed57bbc40ea2B57"
+      address: "0x113169680A417e4dB4AD11832486FF317fB4CB36"
```

Comment:
Can we confirm this is the proxy address? cc @zoruka

Reply:
It's not. should change.